frontend/package.json

```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "classnames": "^2.5.1",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-markdown": "^8.0.6"
```

Comment:
Any reason we don't have the latest version of react-markdown? e.g. `"^9.0.1"`

Reply:
I googled react markdown and the copied the install script from the npm page, hadn't realised that it wasn't the latest, good spot. @dianaPrahoveanu-SL would you mind updating it?